bin/compile

```diff
@@ -13,59 +13,36 @@ unset GIT_DIR
 # parse and derive params
 BUILD_DIR=$1
 CACHE_DIR=$2
-LP_DIR=`cd $(dirname $0); cd ..; pwd`
-BUILDPACK_DIR="$(dirname $(dirname $0))"
-
-function error() {
-  echo " !     $*" >&2
-  exit 1
-}
-
-function indent() {
-  c='s/^/       /'
-  case $(uname) in
-    Darwin) sed -l "$c";;
-    *)      sed -u "$c";;
-  esac
-}
-
-if [[ "${STACK}" == "heroku-16" || "${STACK}" == "heroku-18" || "${STACK}" == "heroku-20" ]]; then
-  PGBOUNCER_VERSION="1.14.0"
-elif [[ "${STACK}" == "cedar-14" ]]; then
-  PGBOUNCER_VERSION="1.7"
-else
-  error "Stack ${STACK} is unsupported!"
-fi
-
-echo "Using pgbouncer version: ${PGBOUNCER_VERSION}" | indent
+BUILDPACK_DIR="$(dirname "$(dirname "$0")")"
 
 # vendor directories
 VENDORED_PGBOUNCER="vendor/pgbouncer"
 
 # vendor pgbouncer into the slug
 PATH="$BUILD_DIR/$VENDORED_PGBOUNCER/bin:$PATH"
-echo "-----> Fetching and vendoring pgbouncer into slug"
+
+mkdir -p "$BUILD_DIR/bin"
 mkdir -p "$BUILD_DIR/$VENDORED_PGBOUNCER"
-tar xzf pgbouncer-${PGBOUNCER_VERSION}-${STACK}.tgz -C ${BUILD_DIR}/${VENDORED_PGBOUNCER}
 
-echo "-----> Moving the configuration generation script into app/bin"
-mkdir -p $BUILD_DIR/bin
-cp "$BUILDPACK_DIR/bin/gen-pgbouncer-conf.sh" $BUILD_DIR/bin
-chmod +x $BUILD_DIR/bin/gen-pgbouncer-conf.sh
+echo "-----> pgbouncer-buildpack: Fetching and vendoring pgbouncer into slug"
```

Comment:
Should this say "Vendoring ..." rather than "Fetching and vendoring" given that the file is not being downloaded as it's already in the buildpack?

Reply:
thx, fixed!